src/OmniSharp.Abstractions/Models/v1/FilesChangedRequest.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+using OmniSharp.Mef;
+
+namespace OmniSharp.Models
+{
+    [OmniSharpEndpoint("/filesChanged", typeof(IEnumerable<Request>), typeof(object), TakeOne = true)]
```

Comment:
So this is worth some discussion.

I did have a static collection before and it worked fairly well, but then I got thinking about how we could have editors extend OmniSharp with extra features that don't necessarily fit as part of OmniSharp proper.

The big example I have in my head, that I'd like to take a stab at in the future, is Continuous Testing. Using the attribute based approach here lets us add arbitrary endpoints for the editor to call through a custom plugin that editor supports or ships with.